deployment/ansible/roles/model-my-watershed.geoprocessing/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Update CA Certificates
+  shell: "update-ca-certificates -f"
```

Comment:
Sounds good. Let me know if you'd like to pair on this, I'm available for that this week. Alternatively, just check out this branch, remove this commit, and follow the testing instructions to see the issue.